File: firstcode/firstassignemt/newassignement/studentscores.py
print("Welcome to student scores evaluator")
score = 0
passed = 0
failed = 0
counter = 1

while counter <= 15:
	score = int(input("Enter a score:  "))
	if score >= 45: 
		passed = passed + 1
	elif score < 45:
		failed = failed + 1
	counter = counter + 1
print(f"{passed} passed the exam and {failed} students failed the exam")

File: firstcode/firstassignemt/myjavaassignment/StudentScores.java
import java.util.Scanner;

public class StudentScores {
	public static void main(String[] args) {
		System.out.println("Welcome to student scores evaluator");
	
	Scanner input = new Scanner(System.in);
	
	int scores = 0;
	int counter = 1;
	int pass = 0;
	int fail = 0;

	while(counter <= 15) {
		System.out.print("Enter a score:  ");
			scores = input.nextInt();
		if(scores >= 45) {
			pass = pass + 1;
		} else if(scores < 45) {
			fail = fail + 1;
		}
	   counter =  counter + 1;	
	}
	System.out.printf("%d student passed the exam and %d students failed the exam %n", pass, fail);
}
}